share/OpenMS/SCRIPTS/ropenms.R

```diff
@@ -0,0 +1,91 @@
+library("reticulate")
+ropenms=import("pyopenms", convert = FALSE)
```

Comment:
can you document on how to fulfill this requirement? i.e. what needs to be build and what additional dependencies do you need?

Reply:
@oliveralka, can you add a line how to deal with the python path problem which @tjeerdijk experienced

Reply:
maybe it would make sense to test for PyOpenMS presence and if not present, output a clear error message that includes Python interpreter used and how the user can install pyOpenMS?